_drafts/2021-01-22-ALTF4-22.md

```diff
@@ -0,0 +1,17 @@
+---
+author: Conor_
+editor: stringweasel, Nanogamer7, Therenas, Firerazer
+image:
+  url: /assets/ALTF4/22/thumbnail.jpg
+  transcription:
+title: "Alt-F4 #22 - Blueprint Rant"
+lang: en
+discuss:
+  discord: https://discord.gg/ceKebbY
+---
+
+For today's 22nd issue of Alt-F4, *Conor_* wanted to share his rather strong opinions on blueprints with us, and why they might prevent you from having the most fun while playing Factorio. We're sure he'd be happy to hear your take on this, so please feel free to share it in the comments!
```

Comment:
Since we don't have comments on the site, would it be better to more explicitly point them towards reddit/the forum?